ccd-client/src/test/java/uk/gov/hmcts/cmc/ccd/client/service/SubmitCaseTest.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.cmc.ccd.client.service;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class SubmitCaseTest {
+    @Before
+    public void setUp() throws Exception {
```

Comment:
What is the point of committing such pointless class? Even if this is WIP makes more sense to include it from commit.

Reply:
It was committed after PR was raised so no point reviewing it as part of this PR. Another PR will be raised for it. It's on branch and WIP.

Reply:
Again, committing test on feature branch makes it part of the PR. How you can say it is not part of PR?